### ISSUE TITLE
Fix Glue::execute to run multiple sql query string

### ISSUE THREAD

**Issue by panarch** [gluesql/gluesql#258]
In `src/glue.rs`, `Glue` module is a kind of wrapper interface which provides users to execute sql queries in a simple way.

It is firstly designed to run multiple sql query string.
e.g. In `https://docs.rs/gluesql/0.7.0/gluesql/`,
```rust
   let sqls = "
         DROP TABLE IF EXISTS Glue;
         CREATE TABLE Glue (id INTEGER);
         INSERT INTO Glue VALUES (100);
         INSERT INTO Glue VALUES (200);
         SELECT * FROM Glue WHERE id > 100;
         DROP TABLE Glue;
     ";

     glue.execute(&sqls).unwrap();
}
```

However, current `Glue` module only execute the first query, (`DROP TABLE IF ..` is only executed)
https://github.com/gluesql/gluesql/blob/b03a0587c22fac5880737026656f0f71e322b500/src/glue.rs#L18-L23

Let's fix this to run multiple sql queries from a single string input.

**Comment by earney** [comment]
Can we close this issue?  It seems to have been resolved in issue #415

**Comment by ever0de** [comment]
> Can we close this issue? It seems to have been resolved in issue #415

This issue has not been resolved.
Please refer to <https://github.com/gluesql/gluesql/issues/415#issuecomment-983711473>.

**Comment by ever0de** [comment]
Killed by <https://github.com/gluesql/gluesql/pull/562>

@ming535  thank you for your contribution
